### DISCUSSION TITLE
Auto generate cuid's from Budibase

### DISCUSSION THREAD

**Discussion by bengribbinpl** [Budibase/budibase#8005] [Help] [answered]
I'm developing an app using Posgresql and Prisma.

Prisma will create default cuid's for each ID in the tables. 

Is it possible for Budibase to generate cuid when creating new entries into the tables?

**Comment by melohagan** [comment, marked as answer]
HI @bengribbinpl 

I don't know anything about Prisma, but if it allows you to add autogenerate constraints to your columns, then whenever you create a row from Budibase, the id would automatically be generated.

This is definitely the case for auto-incrementing ids in Postgres for example.

**Reply by bengribbinpl** [reply]
Hi melohagan,

Thank you for your answer.

Sorry I failed to mention to DB is hosted with Supabase who only do UUID's. The connection is DB <--> Budibase directly.

It's doesn't look like CUID's can be generated on Budibase.

Thanks
